### PR TITLE
[MM-T556] Add e2e test for MM-T556

### PR DESCRIPTION
**Pull request by seongwon-kang**
#### Summary
Add e2e test for Browser tab and team sidebar notification - no unreads/mentions

#### Ticket Link

Github Issues: Fixes https://github.com/mattermost/mattermost-server/issues/15705
Jira Ticket: MM-T556

### CODE REVIEW

**Review by josephbaylon — APPROVED** [review]
LGTM. Test passed locally although the other tests are finicky (I tested on master and they have the same behavior for me locally). Thanks @seongwon-kang for your contribution !!

File: e2e/cypress/integration/notifications/browser_tab_notification_spec.js [hunk 1]
```diff
@@ -96,8 +98,8 @@ describe('Notifications', () => {
             cy.get('.badge').contains('1');
         });
 
-        // * Favicon should stay blue (not turn red) when there are mentions indicated
-        verifyFaviconEquals('favicon-default-16x16.png');
+        // * Favicon should turn red when there are mentions indicated
+        verifyFaviconEquals('favicon-mentions-16x16.png');
```
Comment (seongwon-kang) [inline e2e/cypress/integration/notifications/browser_tab_notification_spec.js:101]: @saturninoabril I moved `Remove mention notification (for initial channel).` to run on before()(ln45), then It worked that we discussed in https://github.com/mattermost/mattermost-server/issues/15703.

**Review by saturninoabril — APPROVED** [review]
Thanks @seongwon-kang! LGTM.

**Pull request merged**